packages/mock-server/src/api/mutations/invoiceLine.ts

```diff
@@ -33,7 +33,7 @@ export const InvoiceLineMutation = {
 
       if (
         invoice.type === InvoiceNodeType.InboundShipment &&
-        invoice.status !== InvoiceNodeStatus.Draft
+        invoice.status !== InvoiceNodeStatus.New
```

Comment:
from here just a bunch of changes to the mock server to support the new statuses